modules/wri_search/wri_search.install

```diff
@@ -43,3 +43,13 @@ function wri_search_update_dependencies() {
 
   return $dependencies;
 }
+
+/**
+ * Adds relevance to the person search, and filters by all phrases.
+ */
+function wri_search_update_9501() {
+  \Drupal::service('distro_helper.updates')->updateConfig('views.view.experts_staff', [
+    'display#default#display_options#filters#search_api_fulltext',
+    'display#default#display_options#sorts#search_api_relevance',
```

Comment:
@mariacha I've used this hook before, but I don't think I've ever seen it in this format. This is not feedback on the PR, just curiosity on my part about what the 'display#...' lines are doing exactly.

Reply:
The # sigh is acting like the array in the yml file, so in practice:
```
  display:
    default:
      display_options:
        filters:
          search_api_fulltext:
            ... whatever ...
        sorts:
            search_api_relevance:
            ... whatever ...
```